_maps/debugdalus.json

```diff
@@ -0,0 +1,8 @@
+{
+    "map_name": "Debugdalus",
+    "map_path": "map_files/debugdalus",
+    "map_file": "tgs_debugdalus.dmm",
+	"traits": [{"Marine Main Ship": false}], 

```

Comment:
I don't think it's necessary to add a negative trait like this since that's already the default.